rainbows.rb

```diff
@@ -17,6 +19,14 @@
   [255,0,125] => "raspberry"
 }
 
+rainbow = [{:red => {r: 255, g: 0, b: 0}}, 
+          {:orange => {r: 255, g: 125, b: 0}}, 
+          {:yellow => {r: 255, g: 255, b: 0}}, 
+          {:green => {r: 0, g: 255, b: 0}}, 
+          {:blue => {r: 0, g: 0, b: 255}}, 
+          {:indigo => {r: 75, g: 0, b: 130}}, 
+          {:violet => {r: 125, g: 0, b: 255}}]
```

Comment:
Better! I'll continue to be nit-picky only because it makes us better :)

The []'s on the array should be on their own lines because it's better for readability and for deleting/commenting out the last line.